src/libs/ReportActionsUtils.js

```diff
@@ -425,6 +425,17 @@ function getLinkedTransactionID(reportID, reportActionID) {
     return reportAction.originalMessage.IOUTransactionID;
 }
 
+/**
+ * Returns the parentReportAction if the given report is a thread/task.
```

Comment:
This comment is not true at all. I don't think we need a description, it's quite clear that the function gets a report action based on the reportID and reportActionID.

Reply:
Updated code for requested changes @neil-marcellini.